app/src/main/java/org/simple/clinic/summary/bloodpressures/BloodPressureSummaryViewUpdate.kt

```diff
@@ -3,21 +3,38 @@ package org.simple.clinic.summary.bloodpressures
 import com.spotify.mobius.Next
 import com.spotify.mobius.Next.noChange
 import com.spotify.mobius.Update
+import org.simple.clinic.facility.FacilityConfig
 import org.simple.clinic.mobius.dispatch
 import org.simple.clinic.mobius.next
+import java.util.UUID
 
-class BloodPressureSummaryViewUpdate : Update<BloodPressureSummaryViewModel, BloodPressureSummaryViewEvent, BloodPressureSummaryViewEffect> {
+class BloodPressureSummaryViewUpdate(
+    private val config: BloodPressureSummaryViewConfig
+) : Update<BloodPressureSummaryViewModel, BloodPressureSummaryViewEvent, BloodPressureSummaryViewEffect> {
   override fun update(
       model: BloodPressureSummaryViewModel,
       event: BloodPressureSummaryViewEvent
   ): Next<BloodPressureSummaryViewModel, BloodPressureSummaryViewEffect> {
     return when (event) {
       is BloodPressuresLoaded -> next(model.bloodPressuresLoaded(event.measurements))
       is BloodPressuresCountLoaded -> next(model.bloodPressuresCountLoaded(event.count))
```

Comment:
Same as the other comment about the observability of the `BloodPressuresCountLoaded` event.